docs/src/components/components/count-status-grid/count-status-grid.config.yml

```diff
@@ -0,0 +1,23 @@
+title: count-status-grid
+status: prototype
+context:
+  entityKind: space
+  status: alt
+  entityCount: '1'
+variants:
+  - name: default
```

Comment:
btw, when there isn't really a default, you can just specify `default: app` to name the app [variant as the default](http://fractal.build/guide/components/variants#the-default-variant)